dgg-utilities.user.js

```diff
@@ -2321,6 +2321,11 @@ function injectScript() {
       let resultNukes;
       let result;
 
+      // exclude whispers sent using slash commands or when the main chatbox is inactive
+      if (text.startsWith("/w ") || text.startsWith("/whisper ") || !$('#chat-windows-select span[title="Destiny GG"].active').length){
```

Comment:
Also, no space before a curly bracket? tonyW

Reply:
made sure to fix this one the hardest